R/conda.R

```diff
@@ -151,6 +152,8 @@ conda_install <- function(envname, packages, forge = TRUE, pip = FALSE, pip_igno
     args <- c("install")
     if (forge)
```

Comment:
```suggestion
    if (forge & ! is.null(channel) )
```
Required to run `conda -c channel` rather than `conda -c -conda-forge -c channel` which returns an error.

Reply:
Are you sure? https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html suggests this should be okay, e.g.

```
$ conda install scipy --channel conda-forge --channel bioconda
```
``

Reply:
I've installed @dimitri-mar's branch in R and running it does not work on my branch unless this change is made. I got an error calling it on a different channel. It seems that in this case, there was not windows build on conda-forge and only on another channel.

Reply:
To clarify, this is required to install "leidenalg" on windows systems (as an example for testing). Running `conda -c conda-forge -c vtraag leidenalg` returns an error. It needs to call `conda -c vtraag leidenalg` as there is no windows build in conda-forge in this case.

Reply:
This is resolved by this PR #752. Thanks! https://github.com/rstudio/reticulate/pull/752/files#diff-ed0960af4336904d09c7d5e7e02472d2R133-R142

I'm having trouble installing it from CRAN or GitHub in version 1.15 though.

```
Error: package or namespace load failed for ‘reticulate’ in get(Info[i, 1], envir = env):
 lazy-load database '/Users/tom/Library/R/3.6/library/reticulate/R/reticulate.rdb' is corrupt
```

As a workaround this installs a working version:

```sh
git clone https://github.com/rstudio/reticulate.git
cd reticulate
```
```r
library("devtools")
document()
install()
```